railib/api.py

```diff
@@ -105,12 +107,14 @@ class Permission(str, Enum):
     "create_user",
     "create_oauth_client",
     "delete_database",
+    "delete_transaction",
```

Comment:
nit: i try to alphabetize lists like this, easier for developers to find what they are looking for. same for code, unless there is an overriding organizational principle.

Reply:
Good point. Done.